spec/factories/master_files.rb

```diff
@@ -57,5 +57,12 @@
         mf.save
       end
     end
+    trait :with_comments do
+      status_code 'COMPLETED'
+      after(:create) do |mf|
+        mf.comment += ['MF Comment 1', 'MF Comment 2']
```

Comment:
Do these have to be added after create?  If they can be added before then the second save could be avoided.

Reply:
I did it this way because that is how we do all the other traits.

Reply:
The other traits in this factory deal with separate fedora objects so they require that the master file be created first to attach to.  This trait is just a property on the master file so is safe to do directly without the after_create.